src/Twig/TwigSortTokenParser.php

```diff
@@ -15,16 +18,25 @@ final class TwigSortTokenParser extends AbstractTokenParser {
   /**
    * {@inheritdoc}
    */
-  public function parse(Token $token): TwigSortSetNode {
-
+  public function parse(Token $token): Node {
+    \trigger_error('The sort tag is deprecated in 3.6.0 and will be removed in 4.x, use the sort_namespaces twig filter.', \E_USER_WARNING);
```

Comment:
I use `E_USER_WARNING` because drush does not handle `E_USER_DEPRECATED` in any way